project/src/main/java/com/google/sps/servlets/CompetitionsServlet.java

```diff
@@ -67,24 +69,139 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     }
   }
 
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String frontEndInfo = getInformationFromFrontEnd(request);
+
+    try {
+      //get specific information from json object
+      JSONObject jsonObj = new JSONObject(frontEndInfo);
+      int userId = jsonObj.getInt("userId");
+      String compName = jsonObj.getString("name");
+      SimpleDateFormat formatter = new SimpleDateFormat("dd-MM-yyyy");
```

Comment:
what does this look like at frontend? Are we dealing with different date formatting options there? (eg do we know this isn't MM-DD-YYYY?)

Reply:
Yes we do. The data that i get from front end is 'dd-mm-yyyy' in String so I'm only converting what I got in String to a date object. But I can try having a look into converting to date object but not specifying the format